pom.xml

```diff
@@ -50,36 +50,36 @@
         <dep.commons.math3>3.6.1</dep.commons.math3>
         <dep.commons.io>2.7</dep.commons.io>
         <dep.guava.version>31.0.1-jre</dep.guava.version>
         <dep.trino.jdbc.version>363</dep.trino.jdbc.version>
-        <dep.jackson.version>2.10.3</dep.jackson.version>
-        <dep.jsonpath.version>2.6.0</dep.jsonpath.version>
-        <dep.junit.version>4.12</dep.junit.version>
+        <dep.jackson.version>2.13.4</dep.jackson.version>
+        <dep.jsonpath.version>2.7.0</dep.jsonpath.version>
+        <dep.junit.version>5.8.2</dep.junit.version>
         <dep.hamcrest.version>2.2</dep.hamcrest.version>
         <dep.hsqldb.version>2.3.4</dep.hsqldb.version>
         <dep.logback.version>1.2.11</dep.logback.version>
-        <dep.postgresql.jdbc.version>9.1-901-1.jdbc4</dep.postgresql.jdbc.version>
-        <dep.slf4j.version>1.7.32</dep.slf4j.version>
-        <dep.jadira.usertype>3.2.0.GA</dep.jadira.usertype>
-        <dep.jackson.jsr310>2.4.0</dep.jackson.jsr310>
-        <dep.jackson.hibernate>2.5.4</dep.jackson.hibernate>
-        <dep.spring.boot.version>1.2.3.RELEASE</dep.spring.boot.version>
-        <dep.spring.retry.version>1.1.2.RELEASE</dep.spring.retry.version>
+        <dep.postgresql.jdbc.version>42.2.8</dep.postgresql.jdbc.version>
+        <dep.slf4j.version>1.7.36</dep.slf4j.version>
+        <dep.spring.boot.version>2.7.3</dep.spring.boot.version>
+        <dep.spring.retry.version>1.3.3</dep.spring.retry.version>
         <dep.freemarker>2.3.22</dep.freemarker>
-        <dep.mockito.version>1.10.19</dep.mockito.version>
-        <dep.javax.validation.api.version>1.1.0.Final</dep.javax.validation.api.version>
+        <dep.mockito.version>4.5.1</dep.mockito.version>
+        <dep.javax.validation.api.version>2.0.1.Final</dep.javax.validation.api.version>
         <dep.javax.annotation.api.version>1.3.2</dep.javax.annotation.api.version>
-        <dep.hibernate.version>4.3.8.Final</dep.hibernate.version>
-        <dep.hibernate.jpa.version>1.0.0.Final</dep.hibernate.jpa.version>
-        <dep.hibernate.validator.version>5.1.3.Final</dep.hibernate.validator.version>
-        <dep.aspectjweaver.version>1.8.6</dep.aspectjweaver.version>
+        <dep.hibernate.version>5.6.10.Final</dep.hibernate.version>
+        <dep.hibernate.jpa.version>1.0.2.Final</dep.hibernate.jpa.version>
+        <dep.hibernate.validator.version>6.1.4.Final</dep.hibernate.validator.version>
+        <dep.aspectjweaver.version>1.9.7</dep.aspectjweaver.version>
         <dep.javax.el>2.2.4</dep.javax.el>
-        <dep.javax.xml.bind.api.version>2.3.2</dep.javax.xml.bind.api.version>
+        <dep.javax.xml.bind.api.version>2.3.3</dep.javax.xml.bind.api.version>
         <dep.testcontainers.version>1.16.0</dep.testcontainers.version>
+        <dep.javassist.version>3.27.0-GA</dep.javassist.version>
+        <dep.byte-buddy.version>1.12.9</dep.byte-buddy.version>
+        <dep.jna.version>5.8.0</dep.jna.version>
```

Comment:
Can we use the latest 5.12.1?

Reply:
Of course we can, it just wasn't required for updating Spring Boot. We should add commits with updates to all libraries, as some versions are from 2015.